feature_development.md

```diff
@@ -0,0 +1,26 @@
+# Feature Development
+
+The feature development playbook is for tracking progress in feature
+development. A Software Engineer or Product Manager might want to track the
+velocity of implemented features in order to capture additional value. This
```

Comment:
Select feature development to understand and share how fast you build.

Reply:
I don't know what my call for action is here. Please be more specific. Should I rewrite something? Which part?

Reply:
This is another case of bad feedback on my part. This is a suggestion for rewrite.